Userland/Libraries/LibWeb/CSS/StyleValues/PlaceSelfStyleValue.cpp

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "PlaceSelfStyleValue.h"
+
+namespace Web::CSS {
+
+ErrorOr<String> PlaceSelfStyleValue::to_string() const
+{
+    return String::formatted("{} {}", TRY(m_properties.align_self->to_string()), TRY(m_properties.justify_self->to_string()));
```

Comment:
The general rule for serialization is, to be as brief as possible. So in cases where these two are the same we should serialize just one value here. eg, `place-self: normal` should serialize as `place-self: normal` instead of `place-self: normal normal`.

Reply:
(We're not good at doing this generally but it's in the spec somewhere.)